lib/Sema/CSDiag.cpp

```diff
@@ -7200,18 +7200,19 @@ bool FailureDiagnosis::visitBindOptionalExpr(BindOptionalExpr *BOE) {
 }
 
 bool FailureDiagnosis::visitIfExpr(IfExpr *IE) {
-  auto typeCheckClauseExpr = [&](Expr *clause) -> Expr * {
-    return typeCheckChildIndependently(clause, Type(), CTP_Unused, TCCOptions(),
+  auto typeCheckClauseExpr = [&](Expr *clause, ContextualTypePurpose convertTypePurpose) -> Expr * {
```

Comment:
@xedin The code is not breaking at any of `diagnoseMemberFailures` within `CSDiag.cpp`. Instead, should I try to add more `tryRawRepresentableFixIts` to a switch statement within `diagnoseContextualConversionError`?